internal/states/encryption/encryptionconfig/config_key_provider.go

```diff
@@ -0,0 +1,35 @@
+package encryptionconfig
```

Comment:
I believe we are missing a test for this file?

Reply:
This is tested from `config_test.go`. There is no explicit test for this file.